### PR TITLE
New commands in Tasker Action "Control Playback": "Skip Forward" and "Skip Back"

### PR DESCRIPTION
**Pull request by joaomgcd**
## Description
Adds ability for the **Control Playback** action in Tasker to skip the number of provided seconds either forward or backward

## Testing Instructions
- Open Tasker
- Add **Pocket Casts Control Playback** action
- Set **Playback Command** to **Skip Forward**
- Set **Time To Skip** to **600**
- Press the **OK** button
- Start playing a podcast in Pocket Casts
- Run the Tasker action and check that it skips 10 minutes forward

## Screenshots or Screencast 
screencapture (16)

## Checklist
- [x] If this is a user-facing change, I have added an entry in CHANGELOG.md
- [x] I have considered whether it makes sense to add tests for my changes
- [x] All strings that need to be localized are in `modules/services/localization/src/main/res/values/strings.xml`
- [x] Any jetpack compose components I added or changed are covered by compose previews
 
#### I have tested any UI changes...

- [x] with different themes
- [x] with a landscape orientation
- [x] with the device set to have a large display and font size
- [x] for accessibility with TalkBack

### CODE REVIEW

**Review by mchowning — COMMENTED** [review]
This works well. Thanks for the contribution @joaomgcd ! 🙇 

There are some formatting issues that are going go fail on CI. You can run `./gradlew spotlessCheck` to see the violations or `./gradlew spotlessApply` to automatically fix them. Also, if you want, you can install our pre-commit hook by running `./gradlew installGitHooks` which runs spotless on every commit. I find the pre-commit hook helpful, but manually running spotless works just fine too.

Long-term I think it would be good if we could tighten up the types in this module a bit so that we're passing around fewer raw Strings (for example, requiring that a skip-to-time be a number at the point of the user's input instead of passing it around as a String). You don't need to worry about that in this PR, but just something to think about as we move forward. I went ahead and added a note about this possible improvement to [the UI improvement issue description](https://github.com/Automattic/pocket-casts-android/issues/471).

File: modules/features/taskerplugin/src/main/java/au/com/shiftyjelly/pocketcasts/taskerplugin/controlplayback/InputControlPlayback.kt [hunk 1]
```diff
@@ -11,13 +11,16 @@ import com.joaomgcd.taskerpluginlibrary.input.TaskerInputRoot
 class InputControlPlayback @JvmOverloads constructor(
     @field:TaskerInputField("command") var command: String? = null,
     @field:TaskerInputField("skipToChapter") var chapterToSkipTo: String? = null,
-    @field:TaskerInputField("timeToSkipToSeconds") var timeToSkipToSeconds: String? = null
+    @field:TaskerInputField("timeToSkipToSeconds") var timeToSkipToSeconds: String? = null,
+    @field:TaskerInputField("timeToSkipSeconds") var timeToSkipSeconds: String? = null
```
Comment (mchowning) [inline modules/features/taskerplugin/src/main/java/au/com/shiftyjelly/pocketcasts/taskerplugin/controlplayback/InputControlPlayback.kt:15]: I wonder if changing these to something shorter like `skipSeconds` and `skipToSeconds` might make it harder to mix these up. The fact that the only difference between them currently is that one has a second "To" in the name makes them look very similar (to my eyes at least). I don't feel strongly though, feel free to leave it as-is if you prefer.

**Comment by mchowning** [comment]
Let's try merging the latest from `main` into this branch after [this PR](https://github.com/Automattic/pocket-casts-android/pull/502) is merged. I think that may resolve the build issues on CI.

**Comment by joaomgcd** [comment]
Awesome! :) Thank you!

**Pull request merged**